### ISSUE TITLE
Screen reader is announcing unnecessary hyperlink information. 

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5014]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader is announcing unnecessary hyperlink information. 

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://airtuserexpdev.azurewebsites.net/Dashboard 

### How to reproduce the issue?

Step 1: Open the URL : [https://airtuserexpdev.azurewebsites.net/Dashboard](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fairtuserexpdev.azurewebsites.net%2fDashboard&data=05%7c01%7cv-shravyar%40microsoft.com%7c09aeb38517334354d48308dbeca14400%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638363949604189146%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=X9RebjeUY64a0z9YqF59KH8oRldy59wbtdE%2fzj0CjvM%3D&reserved=0) in Edge Browser->SAGE chat window will display.
Step 2: Type How do I get a grade C? 
Step 3: Verify whether Screen reader is announcing unnecessary hyperlink information or not.

### What do you expect?

Screen reader should not announce unnecessary hyperlink information.
Ensure: NVDA should announce as 'Accessibility Inventory Report (AIRT) link.'

### What actually happened?

Screen reader is announcing unnecessary hyperlink information.
Observation: NVDA is announcing as 'Accessibility Inventory Report (AIRT) (https://airt.azurewebsites.net/Dashboard)'

### Do you have any screenshots or recordings to repro the issue?

![MAS 1 3 1 - Screen reader is announcing unnecessary hyperlink information](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/a2f06aa6-088e-4d24-9488-9990a08aefd6)

### MAS reference

MAS 1.3.1 – Info and Relationships

**Comment by Kirank915** [comment]
Bug copied from Accenture team: [BUG 11645935](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11645935)

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by compulim** [comment]
We could not access the AIRT dashboard.

Could you copy the HTML code around the hyperlink here?
